Gems/Atom/Tools/AtomToolsFramework/Code/Source/Viewport/RenderViewportWidget.cpp

```diff
@@ -222,12 +222,19 @@ namespace AtomToolsFramework
         switch (event->type()) 
         {
             case QEvent::Resize:
-            case QEvent::ShowToParent: //This event exists to capture the case where a resize event is missed "after" the underlying surface is modified by Qt (Seen during level load in Editor)
+
+            // This event exists to capture the case where a resize event is missed "after" the underlying surface is 
+            // modified by Qt (Seen during level load in Editor)
+            case QEvent::ShowToParent:  
+
+            // Qt is sending this event as the final events after a viewport change. This even needs to trigger a 
+            // resize at this point or else the swapchain won't be resized  to the correct viewport size(s) on vulkan.
+            case QEvent::UpdateLater:   
```

Comment:
Debating whether or not to wrap this with an #if defined (AZ_PLATFORM_LINUX) or not.

Reply:
@spham-amzn Not sure if you saw my comment in  (#9107) because it was closed but the small viewport issue is on  windows Vulkan  so maybe test on that windows to see, otherwise it needs its own fix.

Reply:
@GraphiteSIX Verified that this fixes the issue on Windows as well. Its not just a Linux issue, its a vulkan specific issue